.github/workflows/ci.yml

```diff
@@ -26,8 +26,10 @@ jobs:
       - name: Install dependencies
         run: npm install
```

Comment:
use `npm ci` instead of `npm install` (as `npm install` may mutate the package lock file, but `npm ci` does not)